CHANGELOG.md

```diff
@@ -3,18 +3,20 @@
 <!-- latest_release unreleased -->
 ## Unreleased
 
+<!-- latest_release -->
+
+## [0.74.0](https://github.com/habitat-sh/habitat/tree/0.74.0) (2019-01-28)
+[Full Changelog](https://github.com/habitat-sh/habitat/compare/0.73.0...0.74.0)
+
 #### Behavioral Changes
 - set results folder in PWD not PLAN_CONTEXT [#6087](https://github.com/habitat-sh/habitat/pull/6087) ([mwrock](https://github.com/mwrock))
 
-#### New Features & Enhancements
-- Improve butterfly Server::insert_service [#6053](https://github.com/habitat-sh/habitat/pull/6053) ([baumanj](https://github.com/baumanj))
-
 #### Bug Fixes
 - Fix some clippy lints that were missed due to `make lint` missing some files [#6068](https://github.com/habitat-sh/habitat/pull/6068) ([baumanj](https://github.com/baumanj))
 - Fix &quot;uncontroversial&quot; clippy lints and start making them errors in TravisCI [#6065](https://github.com/habitat-sh/habitat/pull/6065) ([baumanj](https://github.com/baumanj))
-- Simplify insert_service [#6044](https://github.com/habitat-sh/habitat/pull/6044) ([baumanj](https://github.com/baumanj))
-- Only start services that are runnable [#6043](https://github.com/habitat-sh/habitat/pull/6043) ([jamesc](https://github.com/jamesc))
-- Correctly serialize exports in service rumors [#5987](https://github.com/habitat-sh/habitat/pull/5987) ([jamesc](https://github.com/jamesc))
+
+#### New Features & Enhancements
+- Improve butterfly Server::insert_service [#6053](https://github.com/habitat-sh/habitat/pull/6053) ([baumanj](https://github.com/baumanj))
```

Comment:
Can we manually add in #5866?

Reply:
Yeah, I found the PR link and moved it into the Features section. I'll rebase and repush after we merge #6093